packages/core/angular/src/wallet/wallet.types.ts

```diff
@@ -3,7 +3,6 @@ import { PublicKey } from '@solana/web3.js';
 
 export interface WalletState {
     wallets: Wallet[];
-    name: WalletName | null;
```

Comment:
Having the name in the state instead of isolated was leading to weird edge cases, decided to go back and tweak the code. Since this isn't published and it's a private change, I kept the same version.